### PR TITLE
pending  data-testid added

### PR DESCRIPTION
**Pull request by fahad-aot**
# Issue Tracking

JIRA: https://aottech.atlassian.net/browse/FWF-2849
Issue Type: BUG/ FEATURE

# Changes
1.data-testid added for sort icons and save as new version modal

### CODE REVIEW

**Review by abilpraju-aot — APPROVED** [review]

File: forms-flow-web/src/components/Form/Item/View.js [hunk 1]
```diff
@@ -368,7 +368,11 @@ const View = React.memo((props) => {
             onConfirm={props.onConfirm}
           ></SubmissionError>
           {isAuthenticated ? (
-            <Link title={t("Back to Form List")} to={`${redirectUrl}form`}>
+            <Link
+              title={t("Back to Form List")}
+              to={`${redirectUrl}form`}
+              data-testid="back-to-formlist-submission"
```
Comment (shuhaib-aot) [inline forms-flow-web/src/components/Form/Item/View.js:374]: here back to form list right not submission table ?

Comment (shuhaib-aot) [reply forms-flow-web/src/components/Form/Item/View.js:374]: the data test id included submission  that's why

Comment (fahad-aot) [reply forms-flow-web/src/components/Form/Item/View.js:374]: yes. that means back to form list from form submission page .likewise I have added data-testid="back-to-formlist-details" , back to form list from form details page.

Comment (shuhaib-aot) [reply forms-flow-web/src/components/Form/Item/View.js:374]: no need to add details or submision we can keep just back-to-formlist

**Review by shuhaib-aot — CHANGES_REQUESTED** [review]

File: forms-flow-web/src/components/Form/Item/View.js (hunk 1 above)
Comment (fahad-aot) [reply forms-flow-web/src/components/Form/Item/View.js:374]: okay

**Review by abilpraju-aot — APPROVED** [review]

**Review by shuhaib-aot — APPROVED** [review]

**Pull request merged**